### ISSUE TITLE
Convert to TypeScript

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/natural-selection#320]
As a prerequisite to https://github.com/phetsims/natural-selection/issues/319 (Add support for dynamic locale), this sim will be converted to TypeScript.  As noted in https://github.com/phetsims/natural-selection/issues/319#issuecomment-1226082140:

> Adding support for dynamic layout involves changing from string types to Property types, so type-checking would be very helpful. Converting to TypeScript before adding dynamic-locale support is recommended (and approved by @kathy-phet).

**Comment by pixelzoom** [comment]
@kathy-phet change of plans...  I'm not going to convert to this sim to TypeScript.  As I got into it, it because a much larger task that anticipated -- not a couple of days (as I had estimated), but a week or more.   I proceeded with #319 (Add support for dynamic locale) and made good progress.  

Given that I have 7 sims for https://github.com/orgs/phetsims/projects/44 (PhET-iO batch release), I don't think we have time for the TypeScript conversions. 

So... I'm going to close this issue, and we can revist conversion to TypeScript at a later date.
